spec/features/cart/unauthenticated_user_cart_spec.rb

```diff
@@ -11,7 +11,13 @@
           state: true,
           description: "aggressively hopped ipa",
           price: 400}
+          
+  beer2 = {name: "Yeti",
+          state: true,
+          description: "Not aggressively hopped ipa",
+          price: 500}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.